assets/src/hooks/useDetour.ts

```diff
@@ -105,7 +105,7 @@ export const useDetour = ({ routePatternId, shape }: OriginalRoute) => {
   const canUndo = startPoint !== null && state === DetourState.Edit
 
   const undo = () => {
-    if (!canUndo) return
+    // if (!canUndo) return
```

Comment:
This was dead code - `undo` (and `clear`) was never possible to call if `canUndo` wasn't `true`.